cmd/commands/product-release.go

```diff
@@ -37,6 +38,55 @@ type (
 	}
 )
 
+const latest_query = `
+query getProductReleasesList(
+	$productName: String!
+	$filters: ProductReleaseFiltersArgs!
+	$pagination: SlicePaginationArgs
+) {
+	productReleases(productName: $productName, filters: $filters, pagination: $pagination) {
+		edges {
+			node {
+			releaseId
+			steps {
+				environmentName
+				status
+				promotionCommits {
+					applicationId {
+						runtime
+						namespace
+						name
+					}
+					commitSha
+				}
+			}
+			status
+			}
+		}
+	}
+}
+`
+
+const v0_13_4_query = `
```

Comment:
I'll update the specific version references once I've merged the argo-platform changes.